h2o-algos/src/main/java/hex/mojo/api/MojoDelegatingModelHandler.java

```diff
@@ -0,0 +1,22 @@
+package hex.mojo.api;
+
+import hex.mojo.MojoDelegating;
+import hex.mojo.MojoDelegatingModel;
+import hex.mojo.MojoDelegatingModelParameters;
+import water.Job;
+import water.api.Handler;
+
+public class MojoDelegatingModelHandler extends Handler {
+
+    public MojoDelegatingModelV3 createMojoDelegatingModel(int version, MojoDelegatingModelV3 mojoDelegatingModelV3) {
```

Comment:
This is redundant, can be called from R/Python from regular model training methods